### ISSUE TITLE
[Low Viz] WCAG Issues for Low Vision users

### ISSUE THREAD

**Issue by joshueoconnor** [w3c/wcag#78]
The area of low vision a11y and WCAG is a topic that needs attention. 

Some specific areas are:

1)The priority levels of SCs relating to low vision users and the public perception that they are somehow less important.
2) Can we have better more current techniques for developers when building sites/apps for Screen Magnification users? advances in ARIA, being able to move focus etc need to be accommodated in our techniques.
3) Should we revisit our colour contrast algorithm? Should it enforce higher levels of contrast between foreground and background colours? Can we identify instances where ratios need to be higher - headings/ error messages/?
4) Scrolling and screen magnifiers - we need to understand any current problems.
5) Could we have a tutorial for developers to address low vision user needs and also screen magnification users.
6) What supports can UAAG provide on the platform/UA side for Low vision?

[and more from the https://www.w3.org/WAI/GL/wiki/Post_WCAG_2]

Raise SC 1.4.3: Contrast (Minimum) to Level A 
Contrast of Hover and Link Text States
Contrast in Icons

**Comment by marcjohlic** [comment]
Adding in the suggestion to raise [1.4.8 Visual Presentation] (http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-visual-presentation.html) from Level AAA to Level AA for a potential Low Vision extension (and/or Mobile extension).

At a minimum bullet 5 from 1.4.8 to avoid horizontal scrolling:  "Text can be resized without assistive technology up to 200 percent in a way that does not require the user to scroll horizontally to read a line of text on a full-screen window."

**Comment by WayneEDick** [comment]
This outlines some of the need for an LV task force.  This should be held until we have an LVTF.

**Comment by WayneEDick** [comment]
I am sorry about not providing structured mark-up for this post. I don't know how to do it in Github.

It really is time to address the fact that 95% of people with low vision do not use screen magnification to read when they can avoid it.  If screen magnification software was difficult as screen readers one could explain this away as a difficult learning curve, but screen magnification, even advanced features, are simple.  People who do not use screen magnifiers are not too lazy to learn because there is not much to learn. 

Now, reading book sized publications with horizontal strolling is difficult. The difficulty is well documented. If we continue with the wish that screen magnification is adequate or even close to adequate, we will fail most people with low vision.  

It is also important to note that to qualify as having a low vision disability that requires enlarged text anywhere in the world, one requires 300% or greater enlargement.  Thus, while 1.4.8 addresses horizontal scrolling, the enlargement factor is not useful for people classified as having the disability.

Responsive Web Design has proven that re-flowed 400%+ enlargement is possible when industry puts its mind to it. Note: standard print on a device with 1/4 the view port is  equivalent to 400% enlargement. Although RWD is an authoring technique, the resulting responsive technology has proven an effective assistive technology for people with normal vision using mobile devices. Mobile applications have also proven to be excellent assistive technologies for people with normal sight.  The barrier they overcome is small the print and horizontal scrolling required by the need to magnify fixed width web pages and other documents that must be shrunk to fit small screens.  This classification of responsive technology fits the WCAG 2.0 definition of assistive technology: special accommodation (available mobile specific presentation), removal of a barrier (print size well below the critical print size) and a specialized population (visual readers with normal vision). If it can be done for normal readers, it can done for readers with low vision.

Conclusions: 
(1) 1.4.8 should be level A, but it should be broken up. 
(a) Significant enlargement with word wrapping is essential Level A. 
(b) The enlargement factor should be at least 500%, probably 800%. 
Note for (a) and (b): Developers should offer the same technology to users with low vision, that they offer for free to normal readers.
(c) Although easier to implement color control is important and eliminates all issues with contrast. Level A.
(d) The specific character limit per line seems too high for everyone, and does not address the limited line size required for peripheral field loss. This needs work.
(e) The line spacing seems arbitrary, and not really needed in all cases.  The user should be able to specify line spacing, if we are interested in readability. One must consider the impact of line spacing within a large print environment. Maybe 1.3 will do given an enlargement factor of 500%. A smaller line spacing could increase screen capacity by 20%.
(f) User choice for format is more important than fixed sizes.
(g) Finally there are the factors of letter and word spacing as well as font face to be considered.  
(2) The wording of Guideline 1.4 should be changed from "Make it easier for ..." to "Make it possible...". This was expressed in Marc Jolic's observation that somehow people with low vision and partial hearing loss have less need than those with total loss. There is truth to the observation that need is greater for total loss, but when physical barriers prohibit reading or audio understanding the effect is the same.  For most people with partial loss the success criteria make it possible, not easier.  The criteria, the level classifications, and subsequent normative explanations give a very real impression that accessibility for partial loss is just not as important as accessibility for total loss. It also looks like ease of implementation played a more important role than user need. The tone of this guideline needs significant change. 
(3) The use of specific numbers, while precise, looks more like telling developers to encode accommodations more than providing the flexibility necessary for assistive technology. Many SC's in 1.4 read more like accommodations specifications than accessibility criteria. 

This is an extremely well stated observation from Marc Jolic.

**Comment by awkawk** [comment]
Propose closing this issue due to being turned over to the LVTF.
